<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HammingDecodeWidget</class>
 <widget class="QWidget" name="HammingDecodeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="alphLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Путь к файлу с алфавитом</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="decodedTextEdit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>450</y>
     <width>361</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="seqLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>651</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Путь к фалу с декодируемой последовательностью</string>
   </property>
  </widget>
  <widget class="QPushButton" name="alphPushButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Обзор</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="alphLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>651</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextBrowser" name="decodingTextBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>361</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="seqPushButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>320</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Обзор</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="seqLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>651</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tittleLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Декодирование</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="decodePushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>630</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Декодироать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gMatrixPushButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>210</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Обзор</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="gMatrixLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>651</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gMatrixLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Путь к фалу с пораждающей матрицей</string>
   </property>
  </widget>
  <widget class="QLabel" name="decodeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>декодируемая последовательность</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="decodedLabel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>430</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>раскодированная последовательность</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
